vms/shift/views.py

```diff
@@ -442,16 +444,37 @@ def form_valid(self, form):
         shift_id = self.kwargs['shift_id']
         shift = get_shift_by_id(shift_id)
         volunteer_shift = get_volunteer_shift_by_id(volunteer_id, shift_id)
+        volunteer = get_volunteer_by_id(volunteer_id)
         start_time = form.cleaned_data['start_time']
         end_time = form.cleaned_data['end_time']
         shift_start_time = shift.start_time
         shift_end_time = shift.end_time
+        event = shift.job.event
+        site = get_current_site(self.request)
         try:
             if (end_time > start_time):
                 if (start_time >= shift_start_time
                         and end_time <= shift_end_time):
-                    edit_shift_hours(volunteer_id, shift_id, start_time,
-                                     end_time)
+                    request =  form.save(commit=False)
+                    request.volunteer_shift = volunteer_shift
+                    request.save()
+                    message = render_to_string('shift/request_admin.html',
+                                              {'volunteer': volunteer,
+                                               'new_start_time': start_time,
+                                               'new_end_time': end_time,
+                                               'original_start_time': shift_start_time,
+                                               'original_end_time': shift_end_time,
+                                               'event': event,
+                                               'edit_request': request,
+                                               'shift_id': shift_id,
+                                               'domain': site.domain,
+                                               })
+                    try:
+                        send_mail("Edit request", message, "messanger@localhost.com", ["admin@admin.com"])
+                    except:
```

Comment:
Bare exception should never be used. https://www.python.org/dev/peps/pep-0463/#bare-except-clauses
Instead catch `Exception as e`  to catch the exception and raise the exception with its name using `e`
See this for example: https://github.com/systers/postorius/pull/22/files#diff-79124fd10eba4c497247fee3ae8cab01R53